client/src/constants/participantTableConstants.js

```diff
@@ -29,6 +30,10 @@ export const tabs = {
     roles: ['employer', 'health_authority'],
     statuses: ['hired'],
   },
+  'Return Of Service': {
```

Comment:
Both are attached to diff functionality.
- TabStatuses: Filter based on TabSelection to fetch data. The API is less restricted because particular user can have multiple role
- Tab by role: Viewers only restriction based on user most powerful role.
If you want we can discuss this in details.